python/pydantic_core/_pydantic_core.pyi

```diff
@@ -706,25 +706,36 @@ class ValidationError(ValueError):
         Returns:
             The number of errors in the validation error.
         """
-    def errors(self, *, include_url: bool = True, include_context: bool = True) -> list[ErrorDetails]:
+    def errors(
+        self, *, include_url: bool = True, include_context: bool = True, hide_input: bool = False
```

Comment:
I think we should call the parameter `include_input` to be consistent with the other parameters.

Reply:
default is then `True` to match other params too.

Reply:
Makes sense. I'll keep that kind of pattern in mind for the future. Thanks!